docs/asciidoc/faq.adoc

```diff
@@ -85,3 +86,24 @@ Format
 Reactor Netty adds information for the connection at the beginning of every log record (when this is possible).
 Use the id of the connection in order to extract all log records for a particular HTTP request.
 For more information see <<faq.logging-prefix>>
+
+[[faq.connection-closed]]
+== How can I debug "Connection prematurely closed BEFORE response"?
+By default, Reactor Netty clients use a connection pooling. When a connection is acquired from the connection pool, it is checked whether it is still opened.
```

Comment:
Change to:

By default, Reactor Netty clients use connection pooling. When a connection is acquired from the connection pool, it is checked to see whether it is still open.